.github/workflows/upload-windows-zip.yaml

```diff
@@ -0,0 +1,48 @@
+name: Upload windows .zip
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+
+## Needed to upload assets to releases
+permissions:
+  contents: write
+
+jobs:
+  upload-windows-zip:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        include:
+          - target: win32
+          - target: win64
+    env:
+      basename: otp_${{ matrix.target }}_${{ inputs.version }}
+    steps:
+      - name: Install OTP
+        shell: cmd
+        run: |
+          curl.exe --fail -Lo otp.exe https://github.com/erlang/otp/releases/download/OTP-${{ inputs.version }}/${{ env.basename }}.exe
+          otp.exe /S /D=%CD%\${{ env.basename }}
+
+      - name: Create .zip
+        shell: pwsh
+        run: |
+          $root = Get-Location
+
+          cd ${{ env.basename }}
+          rm bin\erl.ini
```

Comment:
Would it make sense to include the sha of the installer in the .zip, so that it is easy to verify the source? Just in case we need to update the installer for some reason...